processors/logout.go

```diff
@@ -25,7 +26,7 @@ func Logout(endpoint string) error {
 
 	// remove token from database
 	if err := auth.Delete(); err != nil {
-		return fmt.Errorf("failed to delete user authentication: %s", err.Error())
+		return util.Errorf("failed to delete user authentication: %s", err.Error())
```

Comment:
Nevermind, I just saw the error implementation below and this would be intentional.